### ISSUE TITLE
Could UD models without generation_config.json affect benchmark results?

### ISSUE THREAD

**Issue by l2dy** [deepsweet/mlx-eval#1]
They forgot to upload `generation_config.json` which specifies the default sampling parameters. Would that affect the results?

https://huggingface.co/unsloth/Qwen3.6-35B-A3B-UD-MLX-4bit/tree/6700c3e5bdeb050a379c8d2a4133f43f3647f20f

**Comment by deepsweet** [comment]
Hi.

No, not at all, since we don't ask a model to generate any new tokens. Instead, we have a long fixed input, and ask a model to make a "forward pass" – compute raw "logits" (probability distributions over the vocabulary) for each token of the input. No temperature, top_p, top_k, etc are involved.

**Comment by l2dy** [comment]
Thanks for the detailed explanation!
